README.md

```diff
@@ -1,5 +1,10 @@
 # jshint loader for webpack
 
+## npm package
+
+``npm install --save jshint-loader jshint``
```

Comment:
Maybe you would be interested in taking https://github.com/webpack/jshint-loader/issues/30?